lib/puppet/reports/satellite.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require __dir__ + '/../util/satellite.rb'
+require "#{__dir__}/../util/satellite.rb"
```

Comment:
this is likely to break the module, see https://github.com/puppetlabs/puppetlabs-registry/pull/287.
opt to use `require_relative`